docs(t&s): update procedures (#4826)
scaleway/docs-content: pages/dedibox-account/how-to/recover-password-lost-email.mdx

### How to recover your password

If you cannot access your e-mail address, please contact our Trust and Safety team with the following information:

> - Indicate your Username

Reviewer: ```suggestion
- Your username
```

- Your current and new e-mail addresses

> - Provide us the telephone number registered on your account

Reviewer: ```suggestion
- The telephone number registered on your account
```

- If you have a individual account, your passport or local ID card
- If you have a company account, the certificate of incorporation and a copy of the last Scaleway invoice you received

> - Submit the complete set of required documents via email to [document@scaleway.com](mailto:document@scaleway.com).

Reviewer: ```suggestion

Submit the complete set of required documents via email to [document@scaleway.com](mailto:document@scaleway.com).
```

### How to recover your password

> If you cannot access your e-mail address, please contact our Trust and Safety team with the following information:

Reviewer 2: ```suggestion
If you cannot access your email address, please contact our Trust and Safety team with the following information:
```

- Your username

> - Your current and new e-mail addresses

Reviewer 2: ```suggestion
- Your current and new email addresses
```

- The telephone number registered on your account

> - If you have a individual account, your passport or local ID card

Reviewer 2: ```suggestion
- If you have an individual account, your passport or local ID card
```

> - If you have a company account, the certificate of incorporation and a copy of the last Scaleway invoice you received

Reviewer 2: ```suggestion
- If you have a corporate account, the certificate of incorporation and a copy of the last Scaleway invoice you received
```
